<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificarMonitor</class>
 <widget class="QWidget" name="ModificarMonitor">
  <property name="windowTitle">
   <string>Modificar / Registrar Monitor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <!-- Añadimos directamente el formulario como layout -->
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelNombre">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditNombre"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelApellido1">
       <property name="text">
        <string>Apellido 1:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditApellido1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelApellido2">
       <property name="text">
        <string>Apellido 2:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditApellido2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelUsuario">
       <property name="text">
        <string>Usuario:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditUsuario"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelContrasena">
       <property name="text">
        <string>Contraseña:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditContrasena">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelEspecialidad">
       <property name="text">
        <string>Especialidad:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditEspecialidad"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditEmail"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelTelefono">
       <property name="text">
        <string>Teléfono:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEditTelefono"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelNCuenta">
       <property name="text">
        <string>Nº Cuenta:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEditNCuenta"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelHorario">
       <property name="text">
        <string>Horario:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEditHorario"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="labelFechaIngreso">
       <property name="text">
        <string>Fecha Ingreso:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="labelFechaIngresoValor">
       <property name="text">
        <string>(No editable)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnGuardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

